store-tests

```diff
@@ -72,7 +72,7 @@ def main():
     statuses = api.all_statuses(opts.revision)
     if not statuses:
         logging.warning("Revision %s has no statuses", opts.revision)
-        return 0
+        sys.exit(1)
```

Comment:
I mean to write here `sys.exit(f"Revision {opts.revision} has no statuses")`

Reply:
Hmm, I really like use `logging` consistently. We get time stamps in the logs and all that. How strongly do you feel about this?

Reply:
Done